pkgs/top-level/all-packages.nix

```diff
@@ -266,6 +266,8 @@ with pkgs;
 
   altserver-linux = callPackage ../tools/misc/altserver-linux { };
 
+  arjun = callPackage ../tools/security/arjun { };
```

Comment:
New packages should go under `pkgs/by-name`.

Reply:
It was already there in the commit

Reply:
gotcha 🙂